lib/rubocop/cop/layout/space_around_operators.rb

```diff
@@ -5,6 +5,17 @@ module Cop
     module Layout
       # Checks that operators have space around them, except for **
       # which should not have surrounding space.
+      #
+      # @example
+      #   # bad
+      #   total = 3*4
+      #   "apple"+"juice"
+      #   my_number = 38/4
+      #
+      #   # good
+      #   total = 3 * 4
+      #   "apple" + "juice"
+      #   my_number = 38 / 4
```

Comment:
Can you add an exception of `**` operator? The cop add an offense if spaces **exist** around `**` operator.

Reply:
I just added that exception beneath the good/bad examples. Not sure if there is a specific way you show exceptions in the docs, happy to update if you prefer a different format.